sale_measure/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_sale_measure,product.measure,model_product_measure,,1,0,0,0
```

Comment:
This security rule are incorrect. We can not write on it and moreover the rule for the object partner.measure is missing.